client/my-sites/site-settings/seo-settings/test/form.jsx

```diff
@@ -31,167 +37,158 @@ const props = {
 };
 
 describe( 'SeoForm basic tests', () => {
+	afterEach( () => {
+		UpsellNudge.mockClear();
+	} );
+
 	test( 'should not blow up and have proper CSS class', () => {
-		const comp = shallow( <SeoForm { ...props } /> );
-		expect( comp.find( '.seo-settings__seo-form' ) ).toHaveLength( 1 );
+		render( <SeoForm { ...props } /> );
+		expect( screen.queryByRole( 'form', { name: /seo site settings/i } ) ).toBeVisible();
 	} );
 
 	test( 'should render conflicted SEO notice when conflictedSeoPlugin is set', () => {
-		const comp = shallow( <SeoForm { ...props } conflictedSeoPlugin={ { name: 'test' } } /> );
-		expect( comp.find( 'Notice' ) ).toHaveLength( 1 );
-		expect( comp.find( 'Notice' ).props().text ).toContain( 'Your SEO settings' );
+		render( <SeoForm { ...props } conflictedSeoPlugin={ { name: 'test' } } /> );
+		const notice = screen.queryByRole( 'status', { name: 'Notice' } );
+		expect( notice ).toBeVisible();
+		expect( notice ).toHaveTextContent( 'Your SEO settings' );
 	} );
 
 	test( 'should not render conflicted SEO notice when conflictedSeoPlugin is not set', () => {
-		const comp = shallow( <SeoForm { ...props } /> );
-		expect( comp.find( 'Notice' ) ).toHaveLength( 0 );
+		render( <SeoForm { ...props } /> );
+		expect( screen.queryByRole( 'status', { name: 'Notice' } ) ).not.toBeInTheDocument();
 	} );
 
 	test( 'should not render Jetpack unsupported notice when is not jetpack site or supports seo', () => {
-		const comp = shallow( <SeoForm { ...props } /> );
-		expect( comp.find( 'Notice' ) ).toHaveLength( 0 );
+		render( <SeoForm { ...props } /> );
+		expect( [] ).toHaveLength( 0 );
```

Comment:
This looks off - did we mean to do this instead:

```suggestion
		expect( screen.queryByRole( 'status', { name: 'Notice' } ) ).not.toBeInTheDocument();
```

Reply:
Yes! Good catch 👍🏻